### ISSUE TITLE
NTR: Deoxypyridinoline to pyridinoline crosslinks ratio increased in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8800]
New term request Deoxypyridinoline to pyridinoline crosslinks ratio increased in urine (segal_020910192813):

**Comment by pnrobinson** [comment]
@MickeySegal
is this actually a synonym of 
[Increased urine deoxypyridinoline level HP:0033355](https://hpo.jax.org/app/browse/term/HP:0033355)?

**Comment by MickeySegal** [comment]
It looks like the cause is a decrease in pyridinoline cross-links, not an increase in deoxypyridinoline cross-links:
https://www.ncbi.nlm.nih.gov/books/NBK1462/
"Deficiency of the enzyme procollagen-lysine, 2-oxoglutarate 5 dioxygenase-1 (PLOD1) results in a deficiency in hydroxylysine-based pyridinoline cross-links in collagens. Detection of an increased ratio of deoxypyridinoline (Dpyr) to pyridinoline (Pyr) cross-links in urine quantitated by high-performance liquid chromatography is a highly sensitive and specific test for PLOD1-related kEDS."

And deoxypyridinoline is a normal component of bone:
https://www.labcorp.com/tests/511105/deoxypyridinoline-dpd-cross-links-serial-monitor
"Deoxypyridinoline (Dpd) is a crosslink of type 1 collagen that provides tensile strength to the collagen matrix of bone. Dpd is released into circulation during bone resorption and is excreted unmetabolized into urine."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000330
- label: Increased ratio of deoxypyridinoline to pyridinoline cross-links in urine
- parents: [Elevated urinary collagen degradation products (HP:0033384)](https://hpo.jax.org/app/browse/term/HP:0033384)
- pmid: PMID:20301635
- def: The relative concentration of deoxypyridinoline (Dpyr) to that of pyridinoline (Pyr) cross-links in urine is above the upper limit of normal.
- comment: Detection of an increased ratio of deoxypyridinoline (Dpyr) to pyridinoline (Pyr) cross-links in urine quantitated by high-performance liquid chromatography is a highly sensitive and specific test for PLOD1-related kyphoscoliotic Ehlers-Danlos syndrome. The normal ratio of Dpyr:Pyr cross-links is approximately 0.2, whereas in kyphoscoliotic EDS the ratio is approximately 6.0.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8800
- synonyms:
